core/includes/drupal.inc

```diff
@@ -2217,6 +2217,16 @@ function drupal_path_alias_whitelist_rebuild($source = NULL) {
   return array();
 }
 
+/**
+ * Rebuild the path alias allowlist.
+ *
+ * @deprecated since 1.16.2
+ */
+function drupal_path_alias_allowlist_rebuild($source = NULL) {
```

Comment:
I added this function because there is a similar issue for Drupal to rename `drupal_path_alias_whitelist_rebuild()` to `drupal_path_alias_allowlist_rebuild()`.

Reply:
I was expecting this to call a function like `backdrop_path_alias_allowlist_rebuild()`. Otherwise maybe we don't this at all.